### PR TITLE
feat: set UNS url checker provider as issuer

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

**Comment by dlecan** [comment]
https://spacelephant.tpondemand.com/entity/2915

File: src/baseCommand.ts [hunk 1]
```diff
@@ -342,9 +342,14 @@ export abstract class BaseCommand extends Command {
...
-        // returns the UNID of UNS forge factory
-        // will be configurable in case of multiple service providers available on UNS network
+    /**
+     * Return the UNID of this service provider depending on command flags
+     */
+    public getServiceProviderUNID(flags: Record<string, any>): string {
+        const urlCheckers: string[] = Managers.configManager.get("network.urlCheckers") || [];
+        if (flags.type === "url" && urlCheckers.length) {
+            return this.unsClientWrapper.network.milestone.urlCheckers[0];
+        }
```
Comment (ftreguer) [inline src/baseCommand.ts]: Why not use `urlCheckers`. What's the difference between `Managers.configManager.get("network.urlCheckers")` and `this.unsClientWrapper.network.milestone.urlCheckers`?

**Review by ftreguer — CHANGES_REQUESTED** [review]

File: src/baseCommand.ts (hunk 1 above)
Comment (peterjah) [reply src/baseCommand.ts]: this is a mistake. unsClientWrapper.network.milestone does not even exists

File: src/baseCommand.ts [hunk 2]
```diff
@@ -342,9 +342,14 @@ export abstract class BaseCommand extends Command {
...
-        // returns the UNID of UNS forge factory
-        // will be configurable in case of multiple service providers available on UNS network
+    /**
+     * Return the UNID of this service provider depending on command flags
+     */
+    public getServiceProviderUNID(flags: Record<string, any>): string {
+        const urlCheckers: string[] = Managers.configManager.get("network.urlCheckers") || [];
+        if (flags.type === "url" && urlCheckers.length) {
+            return Managers.configManager.get("network.urlCheckers")[0];
+        }
```
Comment (dlecan) [inline src/baseCommand.ts:352]: This method could also be overridden only in the dedicated command, no?

**Review by dlecan — CHANGES_REQUESTED** [review]

File: src/baseCommand.ts (hunk 2 above)
Comment (peterjah) [reply src/baseCommand.ts:352]: @dlecan Yes of course but it is used in 2 distincts commands. One heriting directly from baseCommand and another one from propertiesUpdateCommand so the code will be duplicated... what do we prefer ? !

**Review by dlecan — APPROVED** [review]

**Pull request merged**